src/components/header/CouponList.tsx

```
@@ -0,0 +1,74 @@
+                <Percent className="h-5 w-5" />
+            </div>
+            <div>
+                <div className={clsx("text-lg font-bold", coupon.userIssued ? "text-disabled-color" : "text-main-color")}>{coupon.price} 원</div>
+                <div className="text-sm font-medium">{coupon.title}</div>
+            </div>
+            </div>
+            <p className="mt-1 text-xs text-sub-gray">{coupon.content}</p>
+            <p className="mt-0.5 text-xs text-sub-gray">{new Date(coupon.expires).toLocaleString('ko-KR', {year: 'numeric', month: '2-digit', day: '2-digit', hour: '2-digit', minute: '2-digit'}).replace(/\//g, '.').replace(',', '') + ' 만료'}</p>
+            </div>
+            <div className="flex flex-col items-end">
+            <button
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)